delfin/alert_manager/trap_receiver.py

```diff
@@ -33,10 +34,8 @@
 
 LOG = log.getLogger(__name__)
 
-# Currently static mib file list is loaded
-# Mechanism to be changed to load all mib file
-MIB_LOAD_LIST = ['SNMPv2-MIB', 'IF_MIB', 'EMCGATEWAY-MIB', 'FCMGMT-MIB',
-                 'ISM-HUAWEI-MIB']
+# Mib file format to be loaded
+MIB_LOAD_FILE_FORMAT = '.py'
```

Comment:
dont we support if it just *.MIB format?

Reply:
Incase of pysnmp we have 2 options,
1)we can use .mib as it is and do compile using compiler compiler.addMibCompiler function. But it needs external mib path to resolve dependencies. But we may not have external connectivity to compile run time and load it
2) We can take those .mib and convert to .py(precompiled format) offline one time. There are tools such as mibdump toto do this. This .py can be used for loading
we follow second method

Reply:
ok, How user will know he has to compile it before loading? and wich location he has to keep pre compiled files?

Reply:
Path is a config para, we already have it with default as (/var/lib/delfin/mibs) 
For format to be used, we can update doc